runtime/compiler/runtime/MethodMetaData.c

```diff
@@ -333,7 +333,7 @@ static void printMetaData(J9TR_MethodMetaData * methodMetaData)
    for (index = 0; index < iter._stackAtlas->numberOfMaps; ++index)
       {
       getNextMap(&iter, fourByteOffsets);
-      printf("index: %d map address: %p rangeStartOffset: %x rangeEndOffset: %x is inlineMap? %s\n", index, iter._currentMap, iter._rangeStartOffset, iter._rangeEndOffset, IS_BYTECODEINFO_MAP(fourByteOffsets, iter._currentMap) ? "yes" : "no");
+      printf("index: %d map address: %p rangeStartOffset: %zx rangeEndOffset: %zx is inlineMap? %s\n", index, iter._currentMap, iter._rangeStartOffset, iter._rangeEndOffset, IS_BYTECODEINFO_MAP(fourByteOffsets, iter._currentMap) ? "yes" : "no");
```

Comment:
I think that the bullet proof solution for printing uintptr_t is to use `PRIxPTR`. It's ugly, but it's what the standard recommends.

Reply:
Changed to use `OMR_PRIxPTR` like other places in the JIT.